src/rules/property-no-unknown/__tests__/index.js

```diff
@@ -66,6 +66,9 @@ testRule(rule, {
   }, {
     code: ".foo { #{$prop}: black; }",
     description: "ignore property interpolation",
+  }, {
```

Comment:
Could we get *reject* test for an unknown property *within a [custom property set](https://tabatkins.github.io/specs/css-apply-rule/#defining)* please. e.g.

```js
{
  code: ".foo { --custom-property-set: { colr: blue; } }",
  message: messages.rejected("colr"),
  line: 1,
  column: 32,
}
```

Likewise you'll want to check that declarations within custom property sets are *accepted* within `isStandardSyntaxDeclaration-test.js`.

I suspect the code will need to be tweaked to make these tests pass as the conditional currently doesn't seem to distinguish between nested properties and custom property sets.